gef.py

```diff
@@ -10382,6 +10382,14 @@ def __gef_prompt__(current_prompt):
             site.addsitedir(site_packages_dir)
         except FileNotFoundError:
             pass
+        
+        # when Python virtual enviromrnt is used GDB still loads system Python
+        # so GEF doesn't load site-packages dir from environment
+        pythonbin = which("python3")
+        PREFIX = gef_pystring(subprocess.check_output([pythonbin, '-c', 'import os,sys;print((sys.prefix))'])).strip("\\n")
+        if PREFIX != sys.base_prefix:
+            SITE_PACKAGES_DIR = subprocess.check_output([pythonbin, "-c", "import os,sys;print(os.linesep.join(sys.path).strip())"]).decode("utf-8").split()
```

Comment:
```suggestion
            SITE_PACKAGES_DIR = subprocess.check_output(
                [pythonbin, "-c", "import os,sys;print(os.linesep.join(sys.path).strip())"]).decode("utf-8").split()
```
Let's avoid long lines